src/cfnlint/registry.py

```diff
@@ -0,0 +1,87 @@
+"""
+Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: MIT-0
+"""
+
+import json
+import logging
+import os
+import platform
+import subprocess
+from cfnlint.template import Template
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
I didn't find any usage of logger in this file. Maybe we can either remove it or add logs somewhere such as when successfully download a schema?